File: C++/ElHueso.cpp
#include <iostream>

using namespace std;

int main()
{

    int a, b, c, d;

    cin >> a;
    cin >> b;
    cin >> c;
    cin >> d;

    cout << ((a < c && b < d) ? "Hueso 2" : (a > c && b > d) ? "Hueso 1"
                                                             : "Perrito confundido :(");

    return 0;
}

// https://omegaup.com/arena/problem/El-perrito-que-quiere-un-hueso/

File: Python/ElHueso.py

def main(): 
    
    a = int(input())
    b = int(input())
    c = int(input())
    d = int(input())

    if a < c and b < d:
        print("Hueso 2")
    elif a > c and b > d:
        print("Hueso 1")
    else:
        print("Perrito confuncido :( ")

main()

# https://omegaup.com/arena/problem/El-perrito-que-quiere-un-hueso/